.github/actions/npm-license-checker/src/checkLicenses.test.ts

```diff
@@ -153,4 +163,174 @@ describe('checkLicenses', () => {
       plainVertical: true
     })
   })
+
+  it('should handle detailsOutputPath correctly', async () => {
+    options.detailsOutputPath = './license-output.json'
+
+    licenseChecker.init.callsFake((opts: unknown, callback: InitCallback) => {
+      // @ts-expect-error - The first argument can be an error or null
+      callback(null, {})
+    })
```

Comment:
nit: These seems to be repeated in a lot of places. I wonder if it should just be moved to the `beforeEach`